### ISSUE TITLE
Remove ICC named color profiles

### ISSUE THREAD

**Issue by svgeesus** [w3c/csswg-drafts#6095]
Currently, the `color()` function when used with a color profile accepts either numeric parameters or string parameters. The latter is for use with `named color profiles`. These are not well supported (It took me quite a while to find an actual live example for testing), are not allowed in PDF (for CSS to PDF generators) and, for the simple use case of expressing a palette of named colors, CSS Custom Properties aka CSS Variables are an easier and more CSS-like way to produce that palette.

So this is a proposal to remove them from the spec:

- the `color()` property would be simplified, the whole part about string parameters would be removed
- the `@color-profile` would not be allowed to link to them (related discussion at https://github.com/w3c/csswg-drafts/issues/6024#issue-812615639)

Instead I would add an example of using CSS Custom properties to specify a palette of colors.

**Comment by faceless2** [comment]
Great idea.

As I think I've mentioned elsewhere, _if_ we have a named-color ICC profile, we can certainly pull the content out of it and turn it into something we could legitimately put into PDF. But like you, we had to do a proper hunt to find just a single example to test with.

Also as mentioned elsewhere, I think the current syntax is limiting in that it doesn't allow you to specify _how much_ of this special color you want - I can specify a CMYK color with 50% cyan, but the syntax means I can't specify a color that's 50% "Pantone Reflex Blue". So I'm all for dropping this now, before anyone (else) implements it, to allow us to dream up a better solution.

**Comment by svgeesus** [comment]
> before anyone (else) implements it

Yes, sorry and thanks for your [impressive efforts here](https://test.csswg.org/harness/results/css-color-4_dev/grouped/).

> the current syntax is limiting in that it doesn't allow you to specify how much of this special color you want - I can specify a CMYK color with 50% cyan, but the syntax means I can't specify a color that's 50% "Pantone Reflex Blue"

Part of that is getting into [Separation / DeviceN color support](https://github.com/w3c/csswg-drafts/issues/2023) but yes, if we have a way to colorimetrically specify a named color, then we also need a way to mix it with other colors (including white, to get a tint). I hope [`color-mix()`](https://drafts.csswg.org/css-color-5/#color-mix) can help there.

Without knowing things like the dot gain, screening pattern, etc we can't know what measured color `50% "Pantone Reflex Blue"` corresponds to.

**Comment by Crissov** [comment]
Just for the record, #817 was dismissed because what I sought there was (at least in theory) already possible with ICC profiles that specified named colors.

**Comment by svgeesus** [comment]
Yeah, I know, in part. 

Also because you can do it wth CSS Custom Properties, and because of licensing issues and severe under-specifying issues.

**Comment by tabatkins** [comment]
I'm definitely also in support of this simply for the simplification in data model it implies for Typed OM. I'd be happy to work with it if there were good use-cases, but if there's not, let's toss it, please.
